requirements.txt

```diff
@@ -1,4 +1,5 @@
 tornado
 futures
 python-memcached
-psutil
\ No newline at end of file
+psutil
+docker-py
```

Comment:
If you want to go to production with this, you'll need to pin these. Additionally, you should add hashes for them.